calibre-web.json

```
@@ -0,0 +1,51 @@
+{
+  "Calibre Web": {
+    "description": "Calibre-Web is a web app providing a clean interface for browsing, reading and downloading eBooks using an existing Calibre database.<p><strong><em>Note:</em></strong> if using the Calibre Rock-On as database, select the same share used as Media Storage as for the Calibre Rock-On.</p><p>For more information please see <a href='https://github.com/janeczku/calibre-web' target='_blank' rel='noopener noreferrer'>the official Github page</a>. Based on the image provided by Linuxserver.io: <a href='https://hub.docker.com/r/linuxserver/calibre-web' target='_blank' rel='noopener noreferrer'>https://hub.docker.com/r/linuxserver/calibre-web</a>, available for x86-64/amd64, armhf/arm32v7, and arm64/arm64v8.</p>",
```

To align with the image and architecture format.
```suggestion
    "description": "Calibre-Web is a web app providing a clean interface for browsing, reading and downloading eBooks using an existing Calibre database.<p><strong><em>Note:</em></strong> if usig the Calibre Rock-On as database, select the same share used as Media Storage as for the Calibre Rock-On.</p><p>Based on a custom docker image: <a href='https://hub.docker.com/r/linuxserver/calibre-web' target='_blank' rel='noopener noreferrer'>https://hub.docker.com/r/linuxserver/calibre-web target='_blank'>https://hub.docker.com/r/linuxserver/calibre-web</a>, available for amd64 and arm64 architecture.</p>",
```